src/app/App.test.tsx

```diff
@@ -0,0 +1,42 @@
+import {fireEvent, render, screen} from '@testing-library/react'
+import React from 'react'
+import App from './App'
+
+describe('Speech Assistant App', () => {
+  it('should show Consultation Notes button when App is initialized', () => {
+    render(<App />)
+
+    expect(
+      screen.getByRole('button', {
+        name: 'Consultation Notes',
+      }),
+    ).toBeInTheDocument()
+  })
+
+  it('should show Consultation Pad when Consultation Notes button is clicked', () => {
+    render(<App />)
+    const consultationNotesButtonName = {
+      name: 'Consultation Notes',
+    }
+
+    fireEvent.click(screen.getByRole('button', consultationNotesButtonName))
+
+    expect(screen.getByText('Hello This is a div')).toBeInTheDocument()
+    expect(screen.queryByRole('button', consultationNotesButtonName)).toBeNull()
```

Comment:
you could have a negative assertion to validate not.toBeInTheDocument instead of Null

```
expect(
      screen.queryByRole('button', consultationNotesButtonName),
    ).not.toBeInTheDocument()
```